packages/renderer/src/lib/kube/KubeDetailsSummary.svelte

```diff
@@ -0,0 +1,360 @@
+          <td class="py-2 px-4">{pod.spec?.serviceAccountName}</td>
+        </tr>
+        <tr>
+          <td class="py-2 px-4">Restart Policy</td>
+          <td class="py-2 px-4">{pod.spec?.restartPolicy}</td>
+        </tr>
+        <tr>
+          <td class="py-2 px-4">Pod Phase</td>
+          <td class="py-2 px-4">{pod.status?.phase}</td>
+        </tr>
+        <tr>
+          <td class="py-2 px-4">Start Time</td>
+          <td class="py-2 px-4">{pod.status?.startTime}</td>
+        </tr>
+        <tr>
+          <td class="py-2 px-4">Containers</td>
+          <td class="py-2 px-4">{pod.spec?.containers.map(c => c.name).join(', ')}</td>
+        </tr>
+
+        <tr>
+          <td class="py-2 px-4 text-lg pl-1 font-semibold text-purple-400" colspan="2">Networking</td>
+        </tr>
+        <tr>
+          <td class="py-2 px-4">Pod IP</td>
+          <td class="py-2 px-4">{pod.status?.podIP}</td>
+        </tr>
+        <tr>
+          <td class="py-2 px-4">Host IP</td>
+          <td class="py-2 px-4">{pod.status?.hostIP}</td>
+        </tr>
+
+        {#if pod.spec?.containers?.length}
+          <tr>
+            <td class="py-2 px-4 text-lg pl-1 font-semibold text-purple-400" colspan="2">Containers</td>
+          </tr>
+          {#each pod.spec.containers as container}
+            <tr>
+              <td class="py-2 px-4 text-md pl-2 font-semibold text-purple-300" colspan="2">{container.name}</td>
+            </tr>
+            <tr>
+              <td class="py-2 px-4">Name</td>
+              <td class="py-2 px-4">{container.name}</td>
+            </tr>
+            <tr>
+              <td class="py-2 px-4">Image</td>
+              <td class="py-2 px-4">{container.image}</td>
+            </tr>
+            <tr>
+              <td class="py-2 px-4">Image Pull Policy</td>
+              <td class="py-2 px-4">{container.imagePullPolicy}</td>
+            </tr>
+            <tr>
+              <td class="py-2 px-4">Ports</td>
+              <td class="py-2 px-4"
+                >{container.ports?.map(port => `${port.containerPort}/${port.protocol}`).join(', ') || ''}</td>
+            </tr>
+            <tr>
+              <td class="py-2 px-4">Environment Variables</td>
+              <td class="py-2 px-4">
+                {#each container.env ? container.env.map(e => `${e.name}: ${e.value}`) : [] as env}
+                  <div>{env}</div>
+                {/each}
+              </td>
+            </tr>
+            <tr>
+              <td class="py-2 px-4">Volume Mounts</td>
+              <td class="py-2 px-4">{container.volumeMounts?.map(vm => vm.name).join(', ') || ''}</td>
+            </tr>
+            <br />
+          {/each}
+        {/if}
+
+        <!-- Volumes section -->
+        {#if pod.spec?.volumes?.length}
+          <tr>
+            <td class="py-2 px-4 text-lg pl-1 font-semibold text-purple-400" colspan="2">Volumes</td>
+          </tr>
+          {#each pod.spec.volumes as volume}
```

Comment:
I'm wondering if it should not be extracted to its own component as we might to display a volume at different place and it will decrease the lines of Summary widget

another approach would be to have a VolumeUtils, etc where we change PodInfo objects to PodInfoUi, so we don't need logic in svelte components, we just display values

Reply:
You're right! It'll really cut down on KubeDetailsSummary and help us compartmentalize it. I'll update this PR and make a separate test too for it.